### ISSUE TITLE
MO2 GUI freezes when opening mod information

### ISSUE THREAD

**Issue by x0wolfeyez0x** [ModOrganizer2/modorganizer#1743]
MO2 GUI freezes when trying to open *any* mod's information, either by double clicking or by right click > information. Results in a 'not responding' won't which shuts down MO2.

**Comment by MichaelPetersen22** [comment]
I also commonly get this issue. Do you us the *ThisIsWin11* Tool from BuiltbyBel? I found that a setting from that utility (not sure which one) was causing MO2 to have issues. And the only fix was to not run it on my new windows installation

**Comment by x0wolfeyez0x** [comment]
> I also commonly get this issue. Do you use the _ThisIsWin11_ Tool from BuiltbyBel? I found that a setting from that utility (not sure which one) was causing MO2 to have issues. And the only fix was to not run it on my new windows installation

No, I'm not even sure what _ThisIsWin11_ is, so i don't think that's what causing the issue.

**Comment by MichaelPetersen22** [comment]
It's a windows debloating tool to make windows less shit... Good to know that is not the cause though, Or maybe there is multiple causes for this issue.

**Comment by EzioTheDeadPoet** [comment]
The reason in my case for it was the same as with this ["steam_client_freezes_when_i_click_on_install"](https://www.reddit.com/r/Steam/comments/lkg82q/steam_client_freezes_when_i_click_on_install/) issue a network drive that lost it's connection. Not sure why this would affect MO2 or if there is a way to speed this up/ignore the network drive and fix the issue within MO2, but I thought it would be valid information if anyone experiences similar behavior inside MO2.

**Comment by Chef0000** [comment]
After testing Ezio's solution, it was the case for me.  After reconnecting all network drives, I had zero issues.  After then disconnecting one, the problem returned.

**Comment by MrWarmo** [comment]
I have this issue but only on my FO4 install (Portable install) - The skyrim SE install is fine!

I used some sort of windows 11 debloater, I have network drives but are all connected

**Comment by brando56894** [comment]
> The reason in my case for it was the same as with this ["steam_client_freezes_when_i_click_on_install"](https://www.reddit.com/r/Steam/comments/lkg82q/steam_client_freezes_when_i_click_on_install/) issue: a network drive that lost it's connection. Not sure why this would affect MO2 or if there is a way to speed this up/ignore the network drive and fix the issue within MO2, but I thought it would be valid information if anyone experiences similar behavior inside MO2.

Windows is such a terrible OS, I can't believe that this is the reason for the issue. I unmounted my disconnected my network drive and boom, back to normal.
